### DISCUSSION TITLE
In Actions logs the ANSI color gray aka 'bright black' is invisible

### DISCUSSION THREAD

**Discussion by joerick** [community/community#40176] [Mobile]
![image](https://user-images.githubusercontent.com/1244307/204109354-cfb8753e-fd68-42f5-8229-e4ddb0afac75.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.90.0 (84076705)
Device: iPhone 12 Pro (iOS 16.0)
</details>

**Comment by ossanna16** [comment]
Hi there @joerick and welcome to our community! Thank you for reporting this!

**Reply by mattruggio** [reply]
Thx @joerick, this has been added to our mobile iOS issue list.
